YIF.Core.Data/Interfaces/ILectorRepository.cs

```diff
@@ -10,5 +10,6 @@ public interface ILectorRepository<T, K> : IRepository<T, K>
         Task Add(T lector);
         Task<K> GetLectorByUserAndIoEIds(string userId, string ioEId);
         Task<IEnumerable<K>> GetLectorsByIoEId(string ioEId);
+        Task<K> GetLectorByLectorIdAndIoEId(string userId, string ioEId);
```

Comment:
such method already exist, why you created new?

Reply:
Fixed